### PR TITLE
Update aws-auth-guide.md

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
Edited the content based on comments in https://github.com/demisto/etc/issues/27179

## Status
Ready

## Related Issues
fixes: link to the issue

## Description
See comments in https://github.com/demisto/etc/issues/27179

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: content-repo/extra-docs/articles/aws-auth-guide.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ a _Trust Relationship_ and establishes a trusted relationship between two resour
 More information regarding [Trust Relationships can be found here.](https://docs.aws.amazon.com/directoryservice/latest/admin-guide/edit_trust.html)
 
 ### How XSOAR uses STS to Authenticate
+The following describes the EC2 instance metadata use case.
```
Comment (glicht) [inline content-repo/extra-docs/articles/aws-auth-guide.md:35]: Not correct. STS can be used in both EC2 and access key scenarios. Please delete this line.

**Review by glicht — CHANGES_REQUESTED** [review]
Need to move `Prerequisites` to be under: `How XSOAR uses STS to Authenticate`.

**Comment by fvigo** [comment]
@ShirleyDenkberg when you get a chance could you please handle the change Guy suggested, so we can merge it? thanks in advance!

**Comment by glicht** [comment]
@ShirleyDenkberg what's the status here?

**Review by glicht — APPROVED** [review]
I see you added a commit. Looks good.

**Pull request merged**